### PR TITLE
Removed sms_receipt and the corresponding hook. Documented hook_sms_d…

### PR DESCRIPTION
**Pull request by almaudoh**
…elivery_report() in its place.

### CODE REVIEW

File: modules/sms_track/sms_track.module [hunk 1]
```diff
@@ -86,20 +87,16 @@ function sms_track_sms_incoming_preprocess(SmsMessageInterface $sms_message) {
 
 
 /**
- * Implements hook_sms_receipt().
+ * Implements hook_sms_delivery_report().
  *
- * @param string $op
- *   Operation string.
  * @param \Drupal\sms\Message\SmsDeliveryReportInterface[] $reports
```
Comment (dpi) [inline modules/sms_track/sms_track.module:92]: No longer need inline @ param for hooks

File: modules/sms_track/sms_track.module [hunk 2]
```diff
@@ -86,20 +87,16 @@ function sms_track_sms_incoming_preprocess(SmsMessageInterface $sms_message) {
...
  *   Delivery reports received from the SMS gateway.
- * @param array $options
- *   Additional options array.
+ * @param \Symfony\Component\HttpFoundation\Response $response
+ *   The HTTP response that will be sent back to the server.
```
Comment (dpi) [inline modules/sms_track/sms_track.module:95]: -1

Comment (almaudoh) [inline modules/sms_track/sms_track.module:95]: Can you clarify the problem with the comment?

Comment (dpi) [inline modules/sms_track/sms_track.module:95]: No longer need inline @ param for hooks

File: modules/sms_track/sms_track.module (hunk 1 above)
Comment (almaudoh) [inline modules/sms_track/sms_track.module:92]: sms_track has been removed now

**Pull request closed without merging**